src/pkg/whatsapp/whatsapp.go

```diff
@@ -400,7 +413,22 @@ func forwardToWebhook(evt *events.Message) error {
 	if err != nil {
 		return pkgError.WebhookError(fmt.Sprintf("error when create http object %v", err))
 	}
+
+	var secretKey []byte
+	if config.WhatsappWebhookSecret != "" {
+		secretKey = []byte(config.WhatsappWebhookSecret)
+	} else {
+		secretKey = []byte("anything")
```

Comment:
no need if else, use config.WhatsappWebhookSecret with default value if not provided